### ISSUE TITLE
IP grabber

### ISSUE THREAD

**Issue by tioopental** [VedalAI/neurosama.com#13]
Dear everyone

VedalAI can get your IP address. VedalAI can send messages in private DM's to a server of weirdos, and most of your neurosama.com entries are yoinked. Due to this, we are not going to risk both ourselves and another people getting hurt of having their personal information out on the internet because of a VedalAI site. VedalAI and vedal987 can not be trusted. DO NOT add VedalAI and vedal987 on your repos, we are warning you. We are not going to add further details to scare or creep out anybody, but this is true. They have done some weird crap. VedalAI and vedal987 are going to be banned from interwebs due to these reasons: Getting IP addresses and stealing code; VedalAI is doing it in a lower range to keep it lowkey. Now that this is all over the internet, VedalAI  can't keep it "lowkey" anymore. This is for you own good.

With sincerity,
[Your Name]

**Comment by NovaPlayzGames** [comment]
It seems you generated this with ai.

**Comment by Liftyxyz** [comment]
what does bro have against vedal 😭

**Comment by NovaPlayzGames** [comment]
They literally used ai to generated it and did not replace [your name] from it. Also EVERY WEBSITE CAN GET YOUR IP ADDRESS! It is common knowledge if your so scared of it use a vpn oml.

**Comment by gwashark** [comment]
> They literally used ai to generated it and did not replace [your name] from it. Also EVERY WEBSITE CAN GET YOUR IP ADDRESS! It is common knowledge if your so scared of it use a vpn oml.

You are true about this, but since this is static and hosted via GitHub Pages itself, there is no way for vedal to get visit history or all other things unless it’s implemented in the site itself.
